app/models/api/suggestions.php

```diff
@@ -0,0 +1,58 @@
+<?php
+namespace Transvision;
+
+$repositories = ($request->parameters[2] == 'global')
+    ? Project::getRepositories()
+    : [$request->parameters[2]];
+
+$source_strings_merged = [];
+$target_strings_merged = [];
+
+// The search
+$initial_search = Utils::cleanString($request->parameters[5]);
+$terms = Utils::uniqueWords($initial_search);
+
+// Regex options (not currenty used)
+$delimiter = '~';
+$whole_word = isset($check['whole_word']) ? '\b' : '';
+$case_sensitive = isset($check['case_sensitive']) ? '' : 'i';
+$regex = $delimiter . $whole_word . $initial_search . $whole_word .
+         $delimiter . $case_sensitive . 'u';
+
+foreach ($repositories as $repository) {
+    // Loop through all repositories searching in the source locale
```

Comment:
Good point, I never thought about it. In this specific case I moved it inside because there are two cycles, but I think it make sense to have one outside.